anyway/widgets/suburban_widgets/motorcycle_accidents_vs_all_accidents_widget.py

```diff
@@ -163,5 +166,26 @@ def localize_items(request_params: RequestParams, items: Dict) -> Dict:
         }
         return items
 
+    # noinspection PyUnboundLocalVariable
+    def is_included(self) -> bool:
+        vehicle_motorcycle = VehicleCategory.MOTORCYCLE.get_english_display_name()
+        motor_accidents_in_road, motor_accidents_in_all_roads = None, None
+        for item in self.items:
+            if item["label_key"] == location_road:
+                for serie in item["series"]:
+                    if serie["label_key"] == vehicle_motorcycle:
+                        motor_accidents_in_road = serie["value"]
+            elif item["label_key"] == location_all_label:
+                for serie in item["series"]:
+                    if serie["label_key"] == vehicle_motorcycle:
+                        motor_accidents_in_all_roads = serie["value"]
+            else:
+                raise ValueError
+        if not motor_accidents_in_road or not motor_accidents_in_all_roads:
+            raise ValueError

```

Comment:
Hi Shani, I prefer that when en error is detected that the data is printed to the log, to make it easier to understand what happened. E.g. here I think it will make sense to print `self.items`

Reply:
Hi Shani, sorry I didn't make my self clear. I meant to add a log entry in addition to throwing the exception.